.travis.yml

```diff
@@ -2,7 +2,8 @@ language: scala
 jdk:
   - oraclejdk8
```

Comment:
Based on our other comment about only java-client and server targeting 1.6 source compatibility, should we still consider running with jdk6?